homeassistant/components/yeelight/__init__.py

```diff
@@ -241,41 +241,35 @@ def is_ambilight_supported(self) -> bool:
 
     def turn_on(self, duration=DEFAULT_TRANSITION, light_type=None):
         """Turn on device."""
-        import yeelight
-
-        if not light_type:
-            light_type = yeelight.enums.LightType.Main
```

Comment:
you removed this code.

Reply:
Its intentional because it duplicates https://gitlab.com/stavros/python-yeelight/blob/master/yeelight/main.py#L53. I wanted to clean this here for simpler logic and less imports.